### PR TITLE
Ardu-brane

### PR DESCRIPTION
**Pull request by R-Nikhil726**
Added Sensor Ld19 for Lidar Interface with the Pixhawk

### CODE REVIEW

**Comment by R-Nikhil726** [comment]
accept

File: .github/workflows/cmake-single-platform.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This starter workflow is for a CMake project running on a single platform. There is a different starter workflow if you need cross-platform coverage.
```
Comment (Hwurzburg) [inline .github/workflows/cmake-single-platform.yml:1]: remove this!

**Review by Hwurzburg — COMMENTED** [review]
needs additions to build_options.py, and possibly extract_features
needs proper commit message and removal of git related file

**Comment by Hwurzburg** [comment]
nevermind...this is closed

**Pull request closed without merging**